docs/users_guide/landice/test_groups/ismip6_forcing.rst

```diff
@@ -0,0 +1,235 @@
+.. _landice_ismip6_forcing:
+
+ismip6_forcing
+============
```

Comment:
Overall, one thing I think is missing from these instructions for a user is the overall process they would need to run with. Here is my attempt at that info (not formatted correctly for RST).  Please correct me if this info is wrong.

 For a given MALI mesh, a user will need to:
* run `landice/ismip6_forcing/ocean_basal` once, independent of the model, scenario or end year
* run `landice/ismip6_forcing/ocean_thermal` once with `process_obs_data = True` to process the thermal forcing from the observational climatology (used for control runs)
* for each model, scenario and end year:
  * run `landice/ismip6_forcing/atmosphere`
    * If you have never run any model with `process_racmo_smb = True`, you need to do this once
    * for subsequent runs, you can set `process_racmo_smb = False` but it is harmless to run it again as it does nothing if the data is already available.
  * run  `landice/ismip6_forcing/ocean_thermal`